components/server/package.json

```diff
@@ -36,6 +36,7 @@
     "@gitpod/ws-manager": "0.1.5",
     "@google-cloud/storage": "^2.5.0",
     "@octokit/rest": "16.35.0",
+    "agent-base": "4.2.1",
```

Comment:
I had to make this dependency explicit in order to require the proper version of `agent-base` to be patched.